libvast/src/system/node.cpp

```diff
@@ -267,31 +267,30 @@ auto make_component_factories() {
 }
 
 auto make_command_factory() {
+  // When updating this list, remember to update its counterpart in node.cpp as
```

Comment:
Do you mean the counterpart in `application.hpp`?

Reply:
Fixed and force-pushed on both ends.